gdsfactory/geometry/invert.py

```diff
@@ -4,15 +4,14 @@
 import gdsfactory as gf
 from gdsfactory.component import Component
 from gdsfactory.geometry.boolean import boolean
-from gdsfactory.typings import Int2, LayerSpec, Union
+from gdsfactory.typings import LayerSpec
 
 
 @gf.cell
 def invert(
     elements,
     border: float = 10.0,
     precision: float = 1e-4,
-    num_divisions: Union[int, Int2] = (1, 1),
```

Comment:
Is this change intentional? It's not listed in the PR description.

Reply:
this is some deprecated code that does nothing and remained since the gdstk to gdspy transition